### ISSUE TITLE
[sklearn#33842] RFC Add Metadata to HTML display of estimators?

### ISSUE THREAD

**Issue by godofecht** [flooooooooooow/flow#376]
**Mirrored from [scikit-learn#33842](https://github.com/scikit-learn/scikit-learn/issues/33842)**

Original title: RFC Add Metadata to HTML display of estimators?
Original labels: Enhancement, Metadata Routing
Created: 2026-04-23T12:46:52Z

---

Looking at the [metadata routing example](https://scikit-learn.org/dev/auto_examples/miscellaneous/plot_metadata_routing.html), I wonder if it could be useful to add a separate section for Metadata (if enabled and requested) to the estimator displays as well?
WDYT? @adrinjalali @DeaMariaLeon 

---

## Flow context

This issue is mirrored from scikit-learn to explore whether Flow encounters the same problem or can approach it differently. Add analysis below.

**Comment by godofecht** [comment]
Moved to [22](https://github.com/godofecht/flow-scikit/issues/22) in the flow-scikit repository. Further library-specific triage and implementation will happen there.

**Comment by godofecht** [comment]
Moved to [23](https://github.com/godofecht/flow-scikit/issues/23) in the flow-scikit repository. Further library-specific triage and implementation will happen there.
